skl2onnx/operator_converters/calibrated_classifier_cv.py

```diff
@@ -237,6 +236,10 @@ def convert_calibrated_classifier_base_estimator(scope, operator, container,
                 operator.raw_operator.__class__.__name__))
 
     base_model = model.base_estimator
+    if not hasattr(base_model, "predict_proba"):
```

Comment:
Why this check? Also you are checking for predict_proba and error says no decision_function.

Reply:
Fixed